maintainers/maintainer-list.nix

```diff
@@ -6795,6 +6795,10 @@
     githubId = 33523827;
     name = "Harrison Thorne";
   };
+  haruki7049 = {
+    email = "tontonkirikiri@gmail.com";
+    github = "";
```

Comment:
The `github` attribute is missing your github handle. If `github` is specified, `githubId` must be too.